Modules/CalcPerform.lua

```diff
@@ -233,6 +233,14 @@ local function doActorAttribsPoolsConditions(env, actor)
 		modDB:NewMod("EnergyShield", "INC", round(output.Int / 5), "Intelligence")
 	end
 
+	-- Check shrine buffs, must be done before life pool calculated for massive shrine
+	for _, value in ipairs(modDB:List(nil, "ShrineBuff")) do
+		local modList = { value.mod }
+		local inc = modDB:Sum("INC", nil, "BuffEffectOnSelf", "ShrineBuffEffect")
+		local more = modDB:More(nil, "BuffEffectOnSelf", "ShrineBuffEffect")
+		modDB:ScaleAddList(modList, (1 + inc / 100) * more)
```

Comment:
I believe you could do just:
`modDB:ScaleAddList({ value.mod }, calcLib.mod(modDB, nil, "BuffEffectOnSelf", "ShrineBuffEffect")`

This would make it a 1-liner

Reply:
Looked good to me